platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/provider/BuildServices.kt

```diff
@@ -123,6 +125,7 @@ object BuildServices {
             fileCollectionFactory,
             inputFingerprinter,
             gradlePropertiesController,
+            transformFactoryForLegacy
```

Comment:
❓ Why is this necessary?

Reply:
We need it to create an object that does instrumentation.